stablehlo/reference/Tensor.cpp

```diff
@@ -50,6 +51,32 @@ int64_t getSizeInBytes(Type type) {
   report_fatal_error(std::move(err));
 }
 
+// Flattens multi-dimensional index 'index' of a tensor to a linearized
+// index into the underlying storage using the dimension strides `strides_`.
```

Comment:
I don't think the comment is applicable to this function anymore. "to a linearized index into the underlying storage where elements are laid out in canonical order"?